news/backends/sfdc.py

```diff
@@ -30,7 +30,7 @@
     'fsa_school': 'FSA_School__c',
     'fsa_grad_year': 'FSA_Grad_Year__c',
     'fsa_major': 'FSA_Major__c',
-    'fsa_city': 'FSA_City__c',
+    'fsa_city': 'MailingCity',
```

Comment:
Do we need to change `https://github.com/mozilla/basket/pull/209/commits/b3320920830e922099720dfabae06926490a784c#diff-1167109521e8993e37415a37ece27bb1L56` too?